src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
The import `combineSlices` is incorrect. You should use `combineReducers` from `@reduxjs/toolkit` to combine the reducers. Update the import statement to `import { combineReducers, configureStore } from '@reduxjs/toolkit';`.